featurehub-javascript-client-sdk/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+#### 1.1.6
+- Change to interfaces instead of classes for models, and match the OpenAPI current definition
+- Add support for deregistering feature and readiness listeners (ready for React SDK)
+- Deprecate the readyness listener for a readiness listener
```

Comment:
readyness and readiness?

Reply:
because readyness is not valid spelling :-(